test/e2e/kubevirt.go

```diff
@@ -270,19 +270,10 @@ var _ = Describe("Kubevirt Virtual Machines", func() {
 
 		checkEastWestTraffic = func(vmi *kubevirtv1.VirtualMachineInstance, podIPsByName map[string][]string, stage string) {
 			GinkgoHelper()
-			polling := 15 * time.Second
-			timeout := time.Minute
 			for podName, podIPs := range podIPsByName {
 				for _, podIP := range podIPs {
-					output := ""
-					Eventually(func() error {
-						var err error
-						output, err = kubevirt.RunCommand(vmi, fmt.Sprintf("curl http://%s", net.JoinHostPort(podIP, "8000")), polling)
-						return err
-					}).
-						WithPolling(polling).
-						WithTimeout(timeout).
-						Should(Succeed(), func() string { return stage + ": " + podName + ": " + output })
+					output, err := kubevirt.RunCommand(vmi, fmt.Sprintf("curl http://%s", net.JoinHostPort(podIP, "8000")), 5*time.Second)
```

Comment:
So `Eventually` was enmascarading the real issue, since that it's the time it takes for arp neighbors cache to be reconciled, removing it showed the issue and force for it to be solved.